libs/sql-schema-describer/src/postgres.rs

```diff
@@ -417,33 +424,39 @@ impl<'a> super::SqlSchemaDescriberBackend for SqlSchemaDescriber<'a> {
         })
     }
 
-    async fn describe(&self, schema: &str) -> DescriberResult<SqlSchema> {
+    async fn describe(&self, schemas: &[&str]) -> DescriberResult<SqlSchema> {
         let mut sql_schema = SqlSchema::default();
         let mut pg_ext = PostgresSchemaExt::default();
-        let table_names = self.get_table_names(schema, &mut sql_schema).await?;
 
-        self.get_sequences(schema, &mut pg_ext).await?;
-        sql_schema.enums = self.get_enums(schema).await?;
-        self.get_columns(schema, &table_names, &mut sql_schema).await?;
-        self.get_foreign_keys(schema, &table_names, &mut sql_schema).await?;
-        self.get_indices(schema, &table_names, &mut pg_ext, &mut sql_schema)
-            .await?;
+        for schema in schemas {
+            sql_schema.push_namespace((*schema).into());
+        }
+
+        //TODO(matthias) can we get rid of the table names map and instead just use tablewalker_ns everywhere like in get_columns?
+        let table_names = self.get_table_names(&mut sql_schema).await?;
```

Comment:
I think that what is preventing the use of walkers at this point is that they hold a reference to the SqlSchema, so we can't mutate it as long as we have a walker around. That may not be the reason, can't remember exactly.